cmd/lvldb2elastic/config/configParser.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+// Config holds the toml configuration for the levelDB to elastic tool
+type Config struct {
+	General       GeneralConfig       `toml:"general"`
+	ElasticSearch ElasticSearchConfig `toml:"elasticSearch"`
+}
+
+// GeneralConfig holds basic configuration
+type GeneralConfig struct {
+	DBPath  string `toml:"dbPath"`
```

Comment:
why not using the storer config? What happens if we decide to switch from level db to something else?

Reply:
I wouldn't change all the storers' interfaces in order to support returning all the records in the database. Right now I am using the custom structure that you created